### PR TITLE
Make DA monitor handling more generic (part 2)

### PR DESCRIPTION
**Pull request by EdwardSafford-NOAA**
## Description

These changes improve the handling of DA monitor data.  The dimensions and coordinates in use (as specified in the DA monitor control file) are now dynamic lists instead of string literals.  Parsing these lists is now done by dictionaries instead of `if|elif|else` syntax.

Tests have been successfully completed using RadMon time files (2 dimensions), RadMon angle files (3 dimensions), and OznMon time files (1 or 2 dimensions).  

Please don't close issue #85 yet though.  I think there still may be some work to do to handle batch plots.  I'd like to get this reviewed and merged before exploring that (potential) complication.

## Dependencies
None

## Impact
None

Ref #85

### CODE REVIEW

File: src/eva/utilities/config.py
Comment (CoryMartin-NOAA) [inline src/eva/utilities/config.py]: I removed this in #98 as the message is not correct.
This will warn even if the key is found as it is currently written.

**Review by CoryMartin-NOAA — COMMENTED** [review]
One small comment, but otherwise I think this looks good, but want a 2nd pair of eyes to take a look.

File: src/eva/utilities/config.py
Comment (EdwardSafford-NOAA) [reply src/eva/utilities/config.py]: Ok, I've deleted the output.

**Review by CoryMartin-NOAA — APPROVED** [review]
Looks good, but will wait for another approval/our chat this afternoon before merging

**Comment by EdwardSafford-NOAA** [comment]
Hmmm.  Not sure why the Notebook tests have now decided to fail:

`ImportError: cannot import name 'ImportedStyleSheet' from 'bokeh.models' (/home/runner/.local/lib/python3.10/site-packages/bokeh/models/__init__.py)`

**Comment by CoryMartin-NOAA** [comment]
@EdwardSafford-NOAA you're not alone here, this is also in #102 , so it seems like a GitHub actions issue.

**Comment by CoryMartin-NOAA** [comment]
Note, don't worry about the Eva Notebook Tests failure, that will be fixed in #102

**Review by ADCollard — APPROVED** [review]
Looks good to me.

**Pull request merged**
